contracts/contracts/ChatOracle.sol

```diff
@@ -107,6 +107,12 @@ interface IChatGpt {
         string memory errorMessage
     ) external;
 
+    function onOracleKbQueryResponse(
```

Comment:
Not a fan of this name, I think even "onOracleKnowledgeBaseQueryResponse" is better, because I had to think for a second that the Kb means 
But then again this is so long so maybe there is some better option

Reply:
Agreed. I'm not a fan either because I immediately think of a _keyboard_. I'll put the longer version for now.